daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -0,0 +1,94 @@
+---
+type: docs
+title: "Certification lifecycle"
+linkTitle: "Certification lifecycle"
+weight: 200
+description: "The certification lifecycle of the different components available in Dapr"
+---
+
+## Overview
+
+Dapr uses a modular design where functionality is delivered as a component. Each component has an interface definition.  All of the components are pluggable so that in ideal scenarios, you can swap out one component with the same interface for another. Each component that is used in production, needs to matain a certain set of technical requirements that ensure the functional compatibility and robustness of the component.
+
+In general a component needs to be: 
+- compliant with the defined Dapr interfaces 
+- functionally correct and robust
+- well documented and maintained
+
+To make sure a component conforms to the standards set by Dapr, there are a set of tests run against a component in a Dapr maintainers managed environment. Once the tests pass consistently for a certain number of runs, the maturity level can be determined for a component. 
```

Comment:
Please, be precise in "certain number of runs" - there should be a number here. Or remove this requirement.